### PR TITLE
Splunk notable

### PR DESCRIPTION
**Pull request by David-BMS**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/30035
fixes: https://github.com/demisto/etc/issues/33108

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by David-BMS** [comment]
@kirbles19 Please assist as this has to be merged ASAP

File: Packs/SplunkPy/Classifiers/classifier-Splunk_-_Notable_Generic_Incoming_Mapper.json [hunk 1]
```diff
@@ -0,0 +1,188 @@
+{
+    "description": "",
+    "feed": false,
+    "id": "2f7f9005-b1c2-43c3-8bcd-defbe69ec048",
```
Comment (guyfreund) [inline Packs/SplunkPy/Classifiers/classifier-Splunk_-_Notable_Generic_Incoming_Mapper.json:4]:
```suggestion
    "id": "Splunk - Notable Generic Incoming Mapper",
```

File: Packs/SplunkPy/Classifiers/classifier-Splunk_-_Notable_Generic_Outgoing_Mapper.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
+{
+    "description": "",
+    "feed": false,
+    "id": "561b72e1-68b3-426f-89b2-7555afecf1d2",
```
Comment (guyfreund) [inline Packs/SplunkPy/Classifiers/classifier-Splunk_-_Notable_Generic_Outgoing_Mapper.json:4]:
```suggestion
    "id": "Splunk - Notable Generic Outgoing Mapper",
```

File: Packs/SplunkPy/ReleaseNotes/2_1_0.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+
+#### Incident Fields
+- **Notable Drilldown**
+- **Notable Urgency**
+- **Successful Identity Enrichment**
+- **Successful Asset Enrichment**
+- **Notable Owner**
+- **Notable Status**
+- **Successful Drilldown Enrichment**
+- **Notable - ID**
```
Comment (guyfreund) [inline Packs/SplunkPy/ReleaseNotes/2_1_0.md:10]: all successful fields should be gathered together

File: Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+def main():
+    trace = ''
+    try:
+        incident = demisto.incident()
+        # events = incident.get('labels', {}).get('Drilldown', {})
+        # print(incident.get('labels', {}))
+        # print(events)
+        custom_fields = incident.get('CustomFields', {})
+        try:
+            drilldown_results_str = custom_fields.get('assettable', {})
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py:16]:
```suggestion
            asset_results_str = custom_fields.get('assettable', {})
```

File: Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+from CommonServerPython import *  # noqa: F401
+
+
+def main():
+    trace = ''
+    try:
+        incident = demisto.incident()
+        # events = incident.get('labels', {}).get('Drilldown', {})
+        # print(incident.get('labels', {}))
+        # print(events)
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py:13]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+        custom_fields = incident.get('CustomFields', {})
+        try:
+            drilldown_results_str = custom_fields.get('assettable', {})
+            drilldown_results = json.loads(drilldown_results_str)
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py:17]:
```suggestion
            asset_results = json.loads(drilldown_results_str)
```

File: Packs/SplunkPy/Scripts/SplunkShowDrilldown/SplunkShowDrilldown.py [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+from CommonServerPython import *  # noqa: F401
+
+
+def main():
+    trace = ''
+    try:
+        incident = demisto.incident()
+        # events = incident.get('labels', {}).get('Drilldown', {})
+        # print(incident.get('labels', {}))
+        # print(events)
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowDrilldown/SplunkShowDrilldown.py:13]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowDrilldown/SplunkShowDrilldown.py [hunk 8]
```diff
@@ -0,0 +1,54 @@
+import json
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def main():
+    trace = ''
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowDrilldown/SplunkShowDrilldown.py:8]:
```suggestion
```
Please remove all handling of the trace var.

File: Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py [hunk 9]
```diff
@@ -0,0 +1,54 @@
+import json
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def main():
+    trace = ''
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py:8]:
```suggestion
```
Please remove all handling of the trace var.

File: Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+            trace += '8'
+
+        trace += '9'
+
+        return {'ContentsFormat': formats['markdown'], 'Type': entryTypes['note'], 'Contents': markdown}
+    except Exception as exp:
+        return_error('could not parse Splunk events', error=exp)
+    finally:
+        # print('trace: ' + trace)
+        pass
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py:50]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowDrilldown/SplunkShowDrilldown.py [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+            trace += '8'
+
+        trace += '9'
+
+        return {'ContentsFormat': formats['markdown'], 'Type': entryTypes['note'], 'Contents': markdown}
+    except Exception as exp:
+        return_error('could not parse Splunk events', error=exp)
+    finally:
+        # print('trace: ' + trace)
+        pass
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowDrilldown/SplunkShowDrilldown.py:50]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py [hunk 12]
```diff
@@ -0,0 +1,54 @@
+import json
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def main():
+    trace = ''
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py:8]:
```suggestion
```
Please remove all handling of the trace var.

File: Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+    try:
+        incident = demisto.incident()
+        # events = incident.get('labels', {}).get('Drilldown', {})
+        # print(incident.get('labels', {}))
+        # print(events)
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py:13]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+        custom_fields = incident.get('CustomFields', {})
+        try:
+            drilldown_results_str = custom_fields.get('identitytable', {})
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py:16]:
```suggestion
            identity_results_str = custom_fields.get('identitytable', {})
```

File: Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py [hunk 15]
```diff
@@ -0,0 +1,54 @@
...
+            drilldown_results = json.loads(drilldown_results_str)
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py:17]:
```suggestion
            identity_results = json.loads(drilldown_results_str)
```

File: Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py [hunk 16]
```diff
@@ -0,0 +1,54 @@
...
+            trace += '8'
+
+        trace += '9'
+
+        return {'ContentsFormat': formats['markdown'], 'Type': entryTypes['note'], 'Contents': markdown}
+    except Exception as exp:
+        return_error('could not parse Splunk events', error=exp)
+    finally:
+        # print('trace: ' + trace)
+        pass
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py:50]:
```suggestion
```

File: Packs/SplunkPy/pack_metadata.json [hunk 17]
```diff
@@ -29,7 +29,16 @@
...
             "display_name": "Access Investigation"
-        }
+        },
+        "Asset": {
+            "mandatory": true,
+            "display_name": "Asset"
+        },
+        "Identity": {
+        "mandatory": true,
+        "display_name": "Identity"
```
Comment (guyfreund) [inline Packs/SplunkPy/pack_metadata.json:39]: Please fix indentation

**Review by guyfreund — CHANGES_REQUESTED** [review]
Nice 👏 , few notes:
- Remove the `Packs/CommonTypes/ReleaseNotes/2_9_4.md` file
- Remove the `Packs/SplunkPy/Scripts/SplunkShowDrilldown/README.md` file
- Remove the `Packs/SplunkPy/Scripts/SplunkShowAsset/README.md` file
- Remove the `Packs/SplunkPy/Scripts/SplunkShowIdentity/README.md` file
- Please run sdk format on all files in this PR

File: Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py [hunk 18]
```diff
@@ -0,0 +1,47 @@
...
+def main():
+    try:
+        incident = demisto.incident()
+        custom_fields = incident.get('CustomFields', {})
+        try:
+            asset_results_str = custom_fields.get('assettable', {})
+            asset_results = json.loads(drilldown_results_str)
+        except Exception as err:
+            print(f'json error: {str(err)}')
+        trace += '1'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py:16]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py [hunk 19]
```diff
@@ -0,0 +1,47 @@
...
+
+        if not drilldown_results:
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py:18]:
```suggestion
        if not asset_results:
```

File: Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py [hunk 20]
```diff
@@ -0,0 +1,47 @@
...
+            trace += '2'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py:19]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py [hunk 21]
```diff
@@ -0,0 +1,47 @@
...
+            return CommandResults()
+
+        if isinstance(drilldown_results, list):
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py:22]:
```suggestion
        if isinstance(asset_results, list):
```

File: Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py [hunk 22]
```diff
@@ -0,0 +1,47 @@
...
+            trace += '3'
+            events_arr = []
+            for event in drilldown_results:
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py:25]:
```suggestion
            for event in asset_results:
```

File: Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py [hunk 23]
```diff
@@ -0,0 +1,47 @@
...
+        except Exception as err:
+            print(f'json error: {str(err)}')
+        trace += '1'
+
+        if not drilldown_results:
+            trace += '2'
+            return CommandResults()
+
+        if isinstance(drilldown_results, list):
+            trace += '3'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py:23]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py [hunk 24]
```diff
@@ -0,0 +1,47 @@
...
+            events_arr = []
+            for event in drilldown_results:
+                try:
+                    trace += '4'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py:27]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py [hunk 25]
```diff
@@ -0,0 +1,47 @@
...
+                    events_arr.append(event)
+                    trace += '5'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py:29]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py [hunk 26]
```diff
@@ -0,0 +1,47 @@
...
+                except Exception as e:
+                    return_error('json error: ', error=e)
+            markdown = tableToMarkdown("", events_arr, headers=events_arr[0].keys())
+            trace += '6'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py:33]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py [hunk 27]
```diff
@@ -0,0 +1,47 @@
...
+        else:
+            trace += '7'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py:35]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py [hunk 28]
```diff
@@ -0,0 +1,47 @@
...
+            markdown = tableToMarkdown("", drilldown_results)
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py:36]:
```suggestion
            markdown = tableToMarkdown("", asset_results)
```

File: Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py [hunk 29]
```diff
@@ -0,0 +1,47 @@
...
+            trace += '8'
+
+        trace += '9'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowAsset/SplunkShowAsset.py:39]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowDrilldown/SplunkShowDrilldown.py [hunk 30]
```diff
@@ -0,0 +1,47 @@
...
+def main():
+    try:
+        incident = demisto.incident()
+        custom_fields = incident.get('CustomFields', {})
+        try:
+            drilldown_results_str = custom_fields.get('notabledrilldown', {})
+            drilldown_results = json.loads(drilldown_results_str)
+        except Exception as err:
+            print(f'json error: {str(err)}')
+        trace += '1'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowDrilldown/SplunkShowDrilldown.py:16]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowDrilldown/SplunkShowDrilldown.py [hunk 31]
```diff
@@ -0,0 +1,47 @@
...
+
+        if not drilldown_results:
+            trace += '2'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowDrilldown/SplunkShowDrilldown.py:19]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowDrilldown/SplunkShowDrilldown.py [hunk 32]
```diff
@@ -0,0 +1,47 @@
...
+            return CommandResults()
+
+        if isinstance(drilldown_results, list):
+            trace += '3'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowDrilldown/SplunkShowDrilldown.py:23]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowDrilldown/SplunkShowDrilldown.py [hunk 33]
```diff
@@ -0,0 +1,47 @@
...
+            events_arr = []
+            for event in drilldown_results:
+                try:
+                    trace += '4'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowDrilldown/SplunkShowDrilldown.py:27]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowDrilldown/SplunkShowDrilldown.py [hunk 34]
```diff
@@ -0,0 +1,47 @@
...
+                    events_arr.append(event)
+                    trace += '5'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowDrilldown/SplunkShowDrilldown.py:29]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowDrilldown/SplunkShowDrilldown.py [hunk 35]
```diff
@@ -0,0 +1,47 @@
...
+                except Exception as e:
+                    return_error('json error: ', error=e)
+            markdown = tableToMarkdown("", events_arr, headers=events_arr[0].keys())
+            trace += '6'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowDrilldown/SplunkShowDrilldown.py:33]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowDrilldown/SplunkShowDrilldown.py [hunk 36]
```diff
@@ -0,0 +1,47 @@
...
+        else:
+            trace += '7'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowDrilldown/SplunkShowDrilldown.py:35]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowDrilldown/SplunkShowDrilldown.py [hunk 37]
```diff
@@ -0,0 +1,47 @@
...
+            markdown = tableToMarkdown("", drilldown_results)
+            trace += '8'
+
+        trace += '9'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowDrilldown/SplunkShowDrilldown.py:39]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py [hunk 38]
```diff
@@ -0,0 +1,47 @@
...
+        incident = demisto.incident()
+        custom_fields = incident.get('CustomFields', {})
+        try:
+            identity_results_str = custom_fields.get('identitytable', {})
+            identity_results = json.loads(drilldown_results_str)
+        except Exception as err:
+            print(f'json error: {str(err)}')
+        trace += '1'
+
+        if not drilldown_results:
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py:18]:
```suggestion
        if not identity_results:
```

File: Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py [hunk 39]
```diff
@@ -0,0 +1,47 @@
...
+            trace += '2'
+            return CommandResults()
+
+        if isinstance(drilldown_results, list):
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py:22]:
```suggestion
        if isinstance(identity_results, list):
```

File: Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py [hunk 40]
```diff
@@ -0,0 +1,47 @@
...
+            trace += '3'
+            events_arr = []
+            for event in drilldown_results:
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py:25]:
```suggestion
            for event in identity_results:
```

File: Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py [hunk 41]
```diff
@@ -0,0 +1,47 @@
...
+                    trace += '4'
+                    events_arr.append(event)
+                    trace += '5'
+                except Exception as e:
+                    return_error('json error: ', error=e)
+            markdown = tableToMarkdown("", events_arr, headers=events_arr[0].keys())
+            trace += '6'
+        else:
+            trace += '7'
+            markdown = tableToMarkdown("", drilldown_results)
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py:36]:
```suggestion
            markdown = tableToMarkdown("", identity_results)
```

File: Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py [hunk 42]
```diff
@@ -0,0 +1,47 @@
...
+def main():
+    try:
+        incident = demisto.incident()
+        custom_fields = incident.get('CustomFields', {})
+        try:
+            identity_results_str = custom_fields.get('identitytable', {})
+            identity_results = json.loads(drilldown_results_str)
+        except Exception as err:
+            print(f'json error: {str(err)}')
+        trace += '1'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py:16]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py [hunk 43]
```diff
@@ -0,0 +1,47 @@
...
+
+        if not drilldown_results:
+            trace += '2'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py:19]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py [hunk 44]
```diff
@@ -0,0 +1,47 @@
...
+            return CommandResults()
+
+        if isinstance(drilldown_results, list):
+            trace += '3'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py:23]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py [hunk 45]
```diff
@@ -0,0 +1,47 @@
...
+            events_arr = []
+            for event in drilldown_results:
+                try:
+                    trace += '4'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py:27]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py [hunk 46]
```diff
@@ -0,0 +1,47 @@
...
+                    events_arr.append(event)
+                    trace += '5'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py:29]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py [hunk 47]
```diff
@@ -0,0 +1,47 @@
...
+                except Exception as e:
+                    return_error('json error: ', error=e)
+            markdown = tableToMarkdown("", events_arr, headers=events_arr[0].keys())
+            trace += '6'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py:33]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py [hunk 48]
```diff
@@ -0,0 +1,47 @@
...
+        else:
+            trace += '7'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py:35]:
```suggestion
```

File: Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py [hunk 49]
```diff
@@ -0,0 +1,47 @@
...
+            markdown = tableToMarkdown("", drilldown_results)
+            trace += '8'
+
+        trace += '9'
```
Comment (guyfreund) [inline Packs/SplunkPy/Scripts/SplunkShowIdentity/SplunkShowIdentity.py:39]:
```suggestion
```

File: Packs/SplunkPy/pack_metadata.json [hunk 50]
```diff
@@ -29,7 +29,16 @@
...
         }
+
```
Comment (guyfreund) [inline Packs/SplunkPy/pack_metadata.json:41]: Please fix the indentation of the identity pack

**Review by guyfreund — CHANGES_REQUESTED** [review]

**Review by guyfreund — CHANGES_REQUESTED** [review]
Regarding https://app.circleci.com/pipelines/github/demisto/content/87459/workflows/cc5dd501-18ec-4dfb-8c08-2b86112920e0/jobs/361854

Whenever `return_error` is used please raise an exception and wrap the main with `try-except` block and then use `return_error`

**Review by guyfreund — CHANGES_REQUESTED** [review]
The 3 `Show` scripts are a duplicate of code.
We can have just one script `SplunkShowResults` which will get as an argument the key in context to be queried (notabledrilldown, assettable or identitytable).

**Comment by David-BMS** [comment]
@michalgold Please review its ready for merge

**Comment by David-BMS** [comment]
> The 3 `Show` scripts are a duplicate of code.
> We can have just one script `SplunkShowResults` which will get as an argument the key in context to be queried (notabledrilldown, assettable or identitytable).

Added as enhancement here
https://github.com/demisto/etc/issues/37197

Right now will be merged as is

File: Packs/SplunkPy/Layouts/layoutscontainer-Splunk_Notable_Generic.json [hunk 51]
```diff
@@ -0,0 +1,649 @@
...
+                        "moved": false,
+                        "name": "Common CIM Fields",
+                        "static": false,
+                        "w": 2,
+                        "x": 1,
+                        "y": 0
+                    },
+                    {
+                        "displayType": "CARD",
+                        "description": "‘True’ indicates enrichment was successful, even if there were no results found.",
```
Comment (michalgold) [inline Packs/SplunkPy/Layouts/layoutscontainer-Splunk_Notable_Generic.json:489]: ‘True’ indicates enrichment was successful, even if there were no results found. 
See Assets, Identities and Drilldown tabs for details.

File: Packs/CommonTypes/IncidentFields/incidentfield-Occurred_Time.json [hunk 52]
```diff
@@ -0,0 +1,30 @@
+{
+    "associatedToAll": true,
+    "caseInsensitive": true,
+    "cliName": "occurredtime",
```
Comment (michalgold) [inline Packs/CommonTypes/IncidentFields/incidentfield-Occurred_Time.json:4]: why create new field and not use "Occurred"?

**Review by michalgold — APPROVED** [review]

**Review by guyfreund — DISMISSED** [review]

**Pull request merged**